docs/automation.md

```diff
@@ -0,0 +1,24 @@
+Automation is similar to autonomous operation, but occurs during the teleoperated period. The goals of automation are to
+
+1. reduce the load on the human driver and operator,
+2. improve robot capabilities, and
+3. score more points during a match.
+
+# Moving the Robot
+
+Our differential swerve drive (DSD) drive train allows us to rotate and translate the robot at the same time. We automate controls so that the driver controls translation and operator controls rotation. We also smooth the inputs so the robot accelerates and decelerates smoothly.
+
+Other ideas for automation while moving the robot:
+
+- slow down the robot if we're about to drive off the field so we don't slam into walls
+- have waypoints that we can send the robot to by pressing a button (such as moving to the hangar for the endgame)
+- dodging other robots
```

Comment:
Any unexpected motion usually causes confusion for the drivers. It's best to let them dodge the robot.